src/main/java/carsharing/processors/impl/customer/RentNewCarProcessor.java

```diff
@@ -0,0 +1,79 @@
+package carsharing.processors.impl.customer;
+
+import carsharing.ConsoleReader;
+import carsharing.Constants;
+import carsharing.model.Car;
+import carsharing.model.Company;
+import carsharing.model.Customer;
+import carsharing.processors.ICustomerProcessors;
+import carsharing.service.ICarService;
+import carsharing.service.ICompanyService;
+import carsharing.service.ICustomerService;
+
+import java.util.List;
+
+public class RentNewCarProcessor implements ICustomerProcessors {
+
+    private final ICompanyService companyService;
+    private final ICarService carService;
+    private final ICustomerService customerService;
+
+    public RentNewCarProcessor(ICompanyService companyService, ICarService carService, ICustomerService customerService) {
+        this.companyService = companyService;
+        this.carService = carService;
+        this.customerService = customerService;
+    }
+
+    @Override
+    public boolean doActionWithCustomer(int customerId) {
+        List<Company> companiesList = companyService.getAll();
+        if (companiesList.size() == 0) {
+            System.out.println("You didn't rent a car!");
+            return true;
+        }
+
+        Customer customer = customerService.getById(customerId);
+        if (customer.getRentedCarId() > 0) {
```

Comment:
it is more clear to check if renteCarId == null

Reply:
Done

Reply:
I assumed, we discussed it

Reply:
Yes, we discussed it but If I use "!= null" my app does work right, I can rent a car I always get message: ""You've already rented a car!""

Reply:
Done